guides/Retrospectives.md

```diff
@@ -0,0 +1,71 @@
+
+## How to run retrospectives?
+
+__Facilitator__
+A retro needs a facilitator whose role is to:
+
+- Manage time.
+- Assist the group in ordering/organizing items.
+- Engage and encourage discussion (by asking questions/reviewing items).
+- Take notes of action items.
+
+
+Many articles, blog posts, and books suggest having a dedicated facilitator rather than having one of the team members play this role, which would help remove bias. In a small team, that may not be an option but is something for the facilitator to keep in mind.
+
+
+__Format__
+There are dozens of retrospective formats in Agile [Retrospectives](https://www.amazon.com/Agile-Retrospectives-Making-Teams-Great/dp/0977616649) or [online](http://retrospectivewiki.org/index.php?title=Retrospective_Plans). The most commonly used one is called “Glad, Sad, Mad.” According to [Retrospectivewiki](http://retrospectivewiki.org/index.php?title=Agile_Retrospective_Resource_Wiki), this format is described as follows:
+
+
+ > Issues, changes or observations made during a sprint are listed by all participants and then categorized as either Glad, Sad or Mad. These broadly represent positive notes about the sprint/team, negative notes about the sprint/team, and “gripes” (which are often more broad than just actions of the team), respectively. Participants then vote on the issues to be discussed and the issues are then discussed from highest to lowest number of votes until all issues have been discussed or there is no time remaining.
+
+
+1. Everyone gather in a room or zoom call (or both). _*_
+2. Set a timer for 10 minutes to write down items to go in the Glad, Sad, or Mad columns.
+3. Once the timer is finished, group similar items or topics if applicable.
+4. Vote on items that you think should be discussed. _* *_
+5. Go through and discuss each item starting with the item with the most votes.
+6. Generate and assign action items that come out of the discussion.
+7. (Optional) Discuss remaining items that may not have received votes if there is enough time
+
+
+_*_  Make sure to review the action items from the previous retrospective to determine if they were useful. Ideally this should come up as topic items but it may be helpful to specifically call them out when you first start conducting retros.
+
+_* *_ If this is in person typically everyone gets 3-5 votes (represented by stickers) to assign to topics they wish to discuss.
+
+
+__Action Items__
+Be sure to come up with action items that are actionable and can be performed by a team member or the whole team. An example of that might be:
+
+- **Issue or topic discussed**: Sentry React Native, which is supposed to collect crashes, was causing a crash. Entire team has felt this pain point and the team decided to search for an alternative.
+- **Action Item**: Yuki has agreed to do a spike to search for an alternative. This might be a task pulled in next sprint.
```

Comment:
Maybe we should link to a helpful definition of "spike" here?

Reply:
👍 if "spike" is not that common, then we should just reword it.